client/services/tables/tables.service.js

```diff
@@ -6,15 +6,15 @@ angular.module('probedock.tables').factory('tables', function(api) {
       var list = $scope[name] = {
         initialized: false,
         records: [],
-        params: {}
+        params: _.extend({}, options.params)
       };
 
       list.refresh = function(table) {
 
         table.pagination.start = table.pagination.start || 0;
         table.pagination.number = table.pagination.number || options.pageSize || 15;
 
-        var params = _.extend({}, options.params, list.params, {
+        var params = _.extend({}, list.params, {
```

Comment:
Yeps. I moved the options.params to line 9. Without this update, when you receive a link with filters in the query params, the filters are not filled because the params start with empty object. We need to reflect the query params in the table params. Maybe there is a different way ?